packages/experiments/src/components/Stack/docs/StackDos.md

```diff
@@ -0,0 +1,4 @@
+- Use anywhere you would use flexbox.
+- Take into account that the overall performance of the component depends on browser support for flexbox.
```

Comment:
Fabric only supports IE11 and up, which should work fine for flexbox. Maybe say "Stack functionality will degrade if used on non supported Browser".  Also note it's functionality more than performance (speed)